src/main/webapp/view/discussion/discussion.js

```diff
@@ -147,6 +154,30 @@ function updateNewCommentTimestamp(timeMs) {
   newCommentTimestampMs = timeMs;
 }
 
+/**
+ * Returns an array of the `DiscussionComment`s within
+ * `TIME_TOLERANCE_MILLISECONDS` milliseconds to the `timestampMilliseconds`.
```

Comment:
why are these all quoted? Shouldn't you use @code instead?

Reply:
@billie-cat I believe the javascript style prefers ticks to @code, this came up in yesterday sync.

Reply:
See: #171